### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by PeterHaislund** [strandborg/AlsavoCtrl#17] [Show and tell]
Hi Mikko,

I've created a Docker image using the AlsavoCtrl that can be used to quickly setup MQTT publishing.
It pulls your code, compiles it and makes an image that can be started by specifying required parameter.

I use it to publish different states from the heat pump to Home Assistant :)
Thought it might be interesting to others as well.

[Docker Hub](https://hub.docker.com/repository/docker/peterhaislund/alsavo_status)

Peter

**Comment by strandborg** [comment]
Excellent, thanks! I added a link to the Docker image in README.md
